helix-view/src/theme.rs

```diff
@@ -51,8 +63,31 @@ impl Loader {
             self.default_dir.join(filename)
         };
 
+        let path_str = path.to_str();
+
+        // We can return the theme from the cache if it already exists
+        if let Some(path_str) = path_str {
+            let tc = self.theme_cache.lock().unwrap();
+            if tc.contains_key(path_str) {
+                let mut theme = tc.get(path_str).unwrap().clone();
+                theme.name = name.to_string();
+                return Result::Ok(theme);
+            }
+        }
```

Comment:
Do we really need a cache? I think we can probably just read from file directly, if the theme changed running `:theme` again will reload it.